README.md

```diff
@@ -4,9 +4,20 @@ A way to use the gonmea package in Flutter
 
 ## Getting Started
 
-This project is a starting point for a Flutter
-[FFI plugin](https://docs.flutter.dev/development/platform-integration/c-interop),
-a specialized package that includes native code directly invoked with Dart FFI.
+This package contains the Go implementation where it is using the gonmea package to parse nmea readings. To build an app that consumes this package, do the following:
```

Comment:
Do you think it'll useful to link the gonmea package on this line [gonmea](https://github.com/erh/gonmea)

Reply:
yes